module2/src/com/epam/arraysofarrays/TaskLauncher.java

```diff
@@ -0,0 +1,87 @@
+               buildRandomSquareMatrixWithPositiveAndNegativeElements(MATRIX_ORDER));
+
+        //task 2
+        MatrixWorker.findElementsTheMainDiagonal(MatrixInitializer.
+                buildRandomSquareMatrixWithPositiveAndNegativeElements(MATRIX_ORDER));
+
+        //task 3
+        MatrixWorker.printRowAndColumn(MatrixInitializer.
+                buildRandomSquareMatrixWithPositiveAndNegativeElements(MATRIX_ORDER), ROW, COLUMN);
+
+        //task 4
+        System.out.println(Arrays.deepToString(MatrixBuilder.buildMatrixAccordingToPattern(MATRIX_ORDER)));
+
+        //task 5
+        System.out.println(Arrays.deepToString(MatrixBuilder.buildMatrixForTaskFive(MATRIX_ORDER)));
+
+        //task 6
+        System.out.println(Arrays.deepToString(MatrixBuilder.buildMatrixForTaskSix(MATRIX_ORDER)));
+
+        //task 7
+        System.out.println(MatrixBuilder.buildMatrixAndCalculatePositiveElements(MATRIX_ORDER));
+
+        //task 8
+        System.out.println("Enter first column: ");
+        BufferedReader reader = new BufferedReader(new InputStreamReader(System.in));
+        String stringM = reader.readLine();
+        int first = Integer.parseInt(stringM);
+
+        System.out.println("Enter second column: ");
+        String stringN = reader.readLine();
+        int second = Integer.parseInt(stringN);
+        MatrixWorker.swapTwoAnyColumns(MatrixInitializer.
+               buildRandomSquareMatrixWithPositiveElements(MATRIX_ORDER), first, second);
+
+        //task 9
+        MatrixWorker.calculateColumnWithLargestSum(MatrixInitializer.
+                buildRandomSquareMatrixWithPositiveElements(MATRIX_ORDER));
+
+        //task 10
+        System.out.println(Arrays.toString(MatrixWorker.calculatePositiveElementsOfMainDiagonal(MatrixInitializer.
+                buildRandomSquareMatrixWithPositiveAndNegativeElements(MATRIX_ORDER))));
+        //task 11
+        System.out.println(Arrays.deepToString(MatrixWorker.matrixTaskEleven(MatrixInitializer.
+                buildRandomMatrix10x20(10, 20))));
+
+        //task 12 Ascending
+        System.out.println(Arrays.deepToString(MatrixSorter.sortRowsAscendingOrder(MatrixInitializer.
+                buildRandomSquareMatrixWithPositiveAndNegativeElements(MATRIX_ORDER))));
+
+        //task 12 Descending
+        System.out.println(Arrays.deepToString(MatrixSorter.sortRowsDescendingOrder(MatrixInitializer.
+                buildRandomSquareMatrixWithPositiveAndNegativeElements(MATRIX_ORDER))));
+
+        //task 13 Ascending
+        System.out.println(Arrays.deepToString(MatrixSorter.sortColumnsAscendingOrder(MatrixInitializer.
+                buildRandomSquareMatrixWithPositiveAndNegativeElements(MATRIX_ORDER))));
+
+        //task 13 Ascending
+        System.out.println(Arrays.deepToString(MatrixSorter.sortColumnsDescendingOrder(MatrixInitializer.
+                buildRandomSquareMatrixWithPositiveAndNegativeElements(MATRIX_ORDER))));
```

Comment:
You can combine rows and columns methods in one with adding boolean param which specifies ascending or descending